e-vent.ino

```diff
@@ -256,7 +256,7 @@ void loop() {
 
       // Check if patient triggers inhale
       patientTriggered = pressureReader.get() < (pressureReader.peep() - knobs.ac()) 
-          && knobs.ac() > AC_MIN;
+          && (knobs.ac() > AC_MIN - 1e-2);
```

Comment:
@abukhalaf I changed this to remove the unnecessary "f" in the literal to match the style. Let me know if you're good with it, and I'll go ahead and test and merge

Reply:
Both knobs.ac and AC_MIN are floats. Removing the "f" literal causes the complier to treat this as double... a waste of memory. I think the rest of the code need to adopt "f" whenever possible.

Reply:
Actually Arduino Mega doesn't have a larger double type so this will be a float by default. Please see the documentation.
https://www.arduino.cc/reference/en/language/variables/data-types/float/

Reply:
You are right.... that concern is only valid for Arduino Due: https://www.arduino.cc/reference/en/language/variables/data-types/double/